src/utils/columnUtils.ts

```diff
@@ -62,6 +97,14 @@ export function getColumnMetrics<R, SR>(metrics: Metrics<R, SR>): ColumnMetrics<
       resizable: column.resizable ?? metrics.defaultResizable,
       formatter: column.formatter ?? metrics.defaultFormatter
     };
+
+    if (newColumn.rowGroup) {
+      const level = groupBy.push(column.key) - 1;
+      newColumn.groupFormatter = column.groupFormatter ?? ToggleGroupFormatter;
+      newColumn.formatterOptions = newColumn.formatterOptions ?? {};
+      newColumn.formatterOptions.groupFocusable = newColumn.formatterOptions.groupFocusable ?? (row => row.level === level);
```

Comment:
If `formatterOptions` is already defined but `formatterOptions.groupFocusable` isn't, we're setting `groupFocusable` in the original column's `formatterOptions`, which could break things.